cart/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db.models.deletion import CASCADE
+from django.db import models
+
+from ingredient.models import Ingredient
+from user.models import User
+
+class Cart(models.Model):
+    count      = models.IntegerField()
+    user       = models.ForeignKey(User, on_delete=models.CASCADE)
+    ingredient = models.ForeignKey(Ingredient, on_delete=models.CASCADE)
```

Comment:
foreignkey field는 id로 만들어주세요! 
schema가 만들어질 때, column이 user가 아니라 user_id가 되어야 합니다.

```suggestion
class Cart(models.Model):
    user_id        = models.ForeignKey(User, on_delete=models.CASCADE)
    ingredient_id  = models.ForeignKey(Ingredient, on_delete=models.CASCADE)
    count          = models.IntegerField()
```

Reply:
수정완료했습니다.